runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -116,9 +123,10 @@ public synchronized void releaseBuffer() {
     @Override
     public synchronized void acquireBuffer() {
         if (serializedCache == null) {
+            //TODO(Maithem): Use pooled mem
```

Comment:
Hopefully these todos will be done soon.

Reply:
I have another patch that introduces pooled memory. I don't want to do it here just in case there's a bug between the acquireBuffer/releaseBuffer which will create a direct memory leak